src/components/bark/bark-typography.tsx

```diff
@@ -32,12 +32,16 @@ export function BarkH4(props: { children: React.ReactNode }) {
 
 export function BarkH5(props: { children: React.ReactNode }) {
   return (
-    <h5 className="scroll-m-20 text-base font-semibold tracking-tight"></h5>
+    <h5 className="scroll-m-20 text-base font-semibold tracking-tight">
+      {props.children}
+    </h5>
   );
 }
 
-export function BarkP(props: { children: React.ReactNode }) {
+export function BarkP(props: { children: React.ReactNode; classes?: string }) {
   return (
-    <p className="leading-7 [&:not(:first-child)]:mt-6">{props.children}</p>
+    <p className={`${props.classes} leading-7 [&:not(:first-child)]:mt-6`}>
```

Comment:
Why not use `clsx` here?

Reply:
Forgot about this package! Done 1584779139fe1ad5a2c9d78a8f07aedd82f3c69d